ixbrlparse/components/nonnumeric.py

```diff
@@ -11,6 +13,7 @@ def __init__(
         name: str,
         format_: Optional[str],
         value: str,
+        ixt: Optional[str],
```

Comment:
I think if it's optional we should maybe give a default of `None` - this would help with backwards-compatability.